### PR TITLE
feat(*): Support non-root users to deploy clusters

### PR DESCRIPTION
**Pull request by WangMinjie114**
**What type of PR is this?**
/kind feature
/area release

**What this PR does / why we need it**:
支持非 root 用户部署

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @bbbmj @muzi502 

**Does this PR introduce a user-facing change?**:

```release-note
Support non-root users to deploy clusters
```

### CODE REVIEW

File: ansible.cfg [hunk 1]
```diff
@@ -21,3 +21,6 @@ deprecation_warnings=False
 inventory_ignore_extensions = ~, .orig, .bak, .ini, .cfg, .retry, .pyc, .pyo, .creds, .gpg
 [inventory]
 ignore_patterns = artifacts, credentials
+# Support non-root users to deploy clusters
+[privilege_escalation]
+become = true
```
Comment (muzi502) [inline ansible.cfg]: 这里少了个空行，源文件是有个空行的

Comment (muzi502) [reply ansible.cfg]: image

Comment (WangMinjie114) [reply ansible.cfg]: done

**Pull request merged**